processmanager/processinfo.go

```diff
@@ -641,6 +646,19 @@ func (pm *ProcessManager) CleanupPIDs() {
 	}
 }
 
+// NameForPID returns the process name for given PID.
```

Comment:
Depending on how we go about solving #278, we may move these somewhere else. For now, I left them as `processInfo` attributes.